src/Catty/Parser&Serialializer/New Parser & Serializer/Serializer/Context/CBXMLSerializerContext.h

```diff
@@ -58,11 +57,15 @@
 @property (nonatomic, strong) NSMutableDictionary *projectUserListNamePositions;
 
 //------------------------------------------------------------------------------------------------------------
-// ressources data used while traversing the tree
+// resources data used while traversing the tree
 //------------------------------------------------------------------------------------------------------------
 // TODO: refactor this later: remove brickList here and dynamically find brick in scriptList. maybe scripts should be referenced in bricks as well!!
 @property (nonatomic, strong) NSMutableArray *brickList; // contains all bricks (used only by serializer)
 
+@property (nonatomic, strong) Project *project;
+
+- (id)initWithProject:(Project *)project;
```

Comment:
Please make this initializer the `NS_DESIGNATED_INITIALIZER`, and set init and new `NS_UNAVAILABLE`.